packages/core/src/dom_components/model/Component.ts

```diff
@@ -769,10 +771,31 @@ export default class Component extends StyleableModel<ComponentProperties> {
       }
     }
 
-    const attrDataVariable = this.get('attributes-data-variable');
+    const attrDataVariable = this.get('attributes-dynamic-value');
```

Comment:
let's put `attributes-dynamic-value` in a const variable?! 
I think it will be easier to track its usage.